parm/config/gfs/config.ufs

```diff
@@ -171,7 +171,10 @@ case "${fv3_res}" in
         export WRITE_GROUP=2
         export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=10
         export WRITE_GROUP_GFS=4
-        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=20
+        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=10
+	if [[ ${machine} = "HERA" ]]; then
+            export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=20
+	fi 
```

Comment:
Please fix the indentation

Reply:
Why is this just for Hera?  Is this load balanced?

Reply:
I think it was an issue of tabs, this has been updated.

Reply:
> Why is this just for Hera? Is this load balanced?

It's an issue of memory on hera.   The jobs will not run on hera with lower than 20 here and will not run on wcoss2 with 20 here.